app/src/main/java/org/kiwix/kiwixmobile/downloader/DownloadService.java

```diff
@@ -469,6 +481,7 @@ private void updateForeground() {
         }
         // If download is canceled clean up else remove .part from file name
         if (downloadStatus.get(chunk.getNotificationID()) == CANCEL) {
+          Log.i("kiwixdownloadservice", "Download Cancelled, deleting .part file");
```

Comment:
How about moving this log statement a bit lower down in the code and adding the full filename that's being deleted?